stable/kubernetes-dashboard/values.yaml

```diff
@@ -4,8 +4,8 @@
 # name: value
 
 image:
-  repository: k8s.gcr.io/kubernetes-dashboard-amd64
-  tag: v1.10.1
+  repository: kubernetesui/dashboard
+  tag: v2.0.0-beta5
```

Comment:
It's [`v2.0.0-beta6`](https://github.com/kubernetes/dashboard/releases/tag/v2.0.0-beta6)  now 🙂

Reply:
Updated! ;)